src/solver/application.cpp

```diff
@@ -377,6 +377,10 @@ void Application::execute()
         break;
     }
 
+    // Save about-the-study files (comments, notes, etc.)
+    pStudy->setWriter(pResultWriter);
+    pStudy->saveMiscFilesIntoOutput();
```

Comment:
Reminder : 
in case we decide to initialize the writer in the study (and broadcast it in the rest of the program),  the call to **saveMiscFilesIntoOutput()** should be made (if **study.parameters** has the writer's info) where it was originally, that next to call to **prepareOutput()**

Reply:
yes